src/app/project/board/board.component.scss

```diff
@@ -40,8 +41,165 @@ section {
 
 // desktop and tablet devices
 @media (min-width: map-get($grid-breakpoints, phone)) {
-   
+
+  .more-space-bottom {
+    margin-bottom: 15px !important;
```

Comment:
It is old code and I am not sure how it will affect if I remove `!important`. 
@kilchenmann can you tell me if it is safe to remove. It is for desktop and tablet version. If I remove it, I don't see any difference.